ui/main/src/app/business/view/card/card-body.view.spec.ts

```diff
@@ -0,0 +1,315 @@
+        cardBodyView = new CardBodyView(card, userWithPerimeters);
+
+        expect(cardBodyView.isCardAcknowledgedFooterVisible()).toBeTrue();
+    });
+
+    it('GIVEN a card WHEN publisher is an entity AND user is not part of publisher entity THEN acknowledgement footer is not shown', async () => {
+        const userWithPerimeters = getUserWithPerimeters(['ENTITY2']);
+        card = getOneCard({
+            publisher: 'ENTITY1',
+            publisherType: 'ENTITY',
+            process: 'testProcess',
+            processVersion: '1',
+            state: 'state1',
+            entityRecipients: ['ENTITY1', 'ENTITY2', 'ENTITY_FR']
+        });
+        cardBodyView = new CardBodyView(card, userWithPerimeters);
+
+        expect(cardBodyView.isCardAcknowledgedFooterVisible()).toBeFalse();
+    });
+
+    it('GIVEN a card WHEN publisher is an entity AND user is not part of publisher entity AND settings.showAcknowledgmentFooter is true THEN acknowledgement footer is shown', async () => {
+        const userWithPerimeters = getUserWithPerimeters(['ENTITY2']);
+
+        configServerMock.setResponseForWebUIConfiguration(new ServerResponse({}, ServerResponseStatus.OK, null));
+        await firstValueFrom(ConfigService.loadWebUIConfiguration());
+        ConfigService.setConfigValue('settings.showAcknowledgmentFooter', true);
+        card = getOneCard({
+            publisher: 'ENTITY1',
+            publisherType: 'ENTITY',
+            process: 'testProcess',
+            processVersion: '1',
+            state: 'state1',
+            entityRecipients: ['ENTITY1', 'ENTITY2', 'ENTITY_FR']
+        });
+        cardBodyView = new CardBodyView(card, userWithPerimeters);
+
+        expect(cardBodyView.isCardAcknowledgedFooterVisible()).toBeTrue();
+    });
+
+    it('GIVEN a card published by an entity WHEN process.state.showAcknowledgmentFooter == NEVER AND  settings.showAcknowledgmentFooter is true THEN acknowledgement footer is not shown', async () => {
+        const userWithPerimeters = getUserWithPerimeters(['ENTITY1']);
+
+        configServerMock.setResponseForWebUIConfiguration(new ServerResponse({}, ServerResponseStatus.OK, null));
+        await firstValueFrom(ConfigService.loadWebUIConfiguration());
+
+        ConfigService.setConfigValue('settings.showAcknowledgmentFooter', true);
+        card = getOneCard({
+            publisher: 'ENTITY1',
+            publisherType: 'ENTITY',
+            process: 'testProcess',
+            processVersion: '1',
+            state: 'state2',
+            entityRecipients: ['ENTITY1', 'ENTITY2', 'ENTITY_FR']
+        });
+        cardBodyView = new CardBodyView(card, userWithPerimeters);
+
+        expect(cardBodyView.isCardAcknowledgedFooterVisible()).toBeFalse();
+    });
+
+    it('GIVEN a card published by another entity WHEN process.state.showAcknowledgmentFooter == ONLY_FOR_USERS_ALLOWED_TO_EDIT AND user part of entities allowed to edit THEN acknowledgement footer is shown', async () => {
```

Comment:
for state field , we keep the old typo : process.state.showAcknowledgementFooter